frontend/src/styles/tailwind.css

```
@@ -1,4 +1,6 @@
 @import 'tailwindcss' source(none);
 @source '../**/*.{js,ts,jsx,tsx}';
 
-@import 'tw-animate-css';
\ No newline at end of file
+@import 'tw-animate-css';
```

Stylelint (17.10.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)